cms/djangoapps/contentstore/views/preview.py

```diff
@@ -265,6 +266,16 @@ def _is_xblock_reorderable(xblock, context):
         return False
 
 
+def _is_xblock_movable(xblock):
+    """
+    Returns true if the specified xblock can be moved otherwise returns false.
+    """
+    return not (
+        isinstance(xblock.location, LibraryUsageLocator) or
+        (xblock.parent and xblock.parent.category == 'library_content')
+    )
```

Comment:
Let's go back to previous approach and do not put specific check to general place.